platform-api/che-core-api-git/src/main/java/org/eclipse/che/api/git/GitService.java

```diff
@@ -391,6 +388,17 @@ public void tagDelete(TagDeleteRequest request) throws ApiException {
                     String[] keyValues = row.split("=", 2);
                     result.put(keyValues[0], keyValues[1]);
                 }
+            } else if (request.isSet()) {
```

Comment:
I think you should create a separate get api method with query parameter for this. It looks very strange doing `config.set` in method that called `getConfig`.